integrations/cdi/jedis-cdi/README.adoc

```diff
@@ -79,12 +79,87 @@ redis.clients.jedis.JedisPool.orders.port
-ultimately in a `JedisPool` object injectable under the name `orders`
-with a `port` property set to `6379` like so:
+ultimately in a `JedisPool` object injectable under the instance name
+of `orders` with a `port` property set to `6379` like so:
 
 [source,java]
 ----
 @Inject
 @Named("orders")
 private JedisPool pool;
 ----
+
+Property names that start with
+ `redis.clients.jedis.JedisPool.instanceName.` are parsed, and the
+ remaining portion of each name is treated as a Java Bean property of
+ https://static.javadoc.io/redis.clients/jedis/2.9.0/redis/clients/jedis/JedisPool.html[`JedisPool`],
+ or as a primitive value accepted by
+ https://static.javadoc.io/redis.clients/jedis/2.9.0/redis/clients/jedis/JedisPool.html#JedisPool-org.apache.commons.pool2.impl.GenericObjectPoolConfig-java.lang.String-int-int-java.lang.String-int-boolean-javax.net.ssl.SSLSocketFactory-javax.net.ssl.SSLParameters-javax.net.ssl.HostnameVerifier-[its
+ most specific constructor].  Because the `JedisPool` class inherits
+ from the
+ https://static.javadoc.io/redis.clients/jedis/2.9.0/redis/clients/util/Pool.html[`Pool`]
+ class, its writable Java Bean properties are available as well.
+
+Accordingly, the `JedisPool` properties that can be set are as
+follows, where `instanceName` should be replaced with the actual name
+used in application code:
+
+* `redis.clients.jedis.JedisPool.instanceName.clientName`
+* `redis.clients.jedis.JedisPool.instanceName.connectionTimeout`
+* `redis.clients.jedis.JedisPool.instanceName.database`
+* `redis.clients.jedis.JedisPool.instanceName.host`
+* `redis.clients.jedis.JedisPool.instanceName.password`
+* `redis.clients.jedis.JedisPool.instanceName.port`
+* `redis.clients.jedis.JedisPool.instanceName.socketTimeout`
+* `redis.clients.jedis.JedisPool.instanceName.ssl`
+
+Any documentation for these properties that exists may be found in the
+javadocs for the
+https://static.javadoc.io/redis.clients/jedis/2.9.0/redis/clients/jedis/JedisPool.html[`JedisPool`]
+and
+https://static.javadoc.io/redis.clients/jedis/2.9.0/redis/clients/util/Pool.html[`Pool`]
+classes.
+
+For advanced configuration use cases, it is also possible to configure
+the
+https://static.javadoc.io/redis.clients/jedis/2.9.0/redis/clients/jedis/JedisPoolConfig.html[`JedisPoolConfig`]
+object used by the Helidon Jedis CDI Integration project to create
+`JedisPool` instances.  In this case,
+https://static.javadoc.io/org.eclipse.microprofile.config/microprofile-config-api/1.3/org/eclipse/microprofile/config/Config.html#getPropertyNames--[property
+names] that start with
+`redis.clients.jedis.JedisPoolConfig.`_instanceName_`.` will be
+parsed, and the remaining portion of each such name will be treated as
+a Java Bean or constructor property of `JedisPool` (or of its
+superclasses).
+
+Accordingly, the `JedisPoolConfig` Java Bean properties that can be
+set are as follows, where `instanceName` should be replaced with the
+actual named used in application code:
+
```

Comment:
Accordingly, the following are the `JedisPoolConfig` Java Bean properties that can be set. Replace `instanceName` with the actual name used in the application code: